apps/web/lib/app-providers.tsx

```diff
@@ -101,6 +101,17 @@ const CalcomThemeProvider = (
       forcedTheme={forcedTheme}
       storageKey={storageKey}
       attribute="class">
+      {/* Embed Mode can be detected reliably only on client side here as there can be static generated pages as well which can't determine if it's embed mode at backend */}
+      {/* color-scheme makes background:transparent not work in iframe which is required by embed. */}
+      {typeof window !== "undefined" && !isEmbedMode && (
+        <style jsx global>
```

Comment:
At server embed can be false(even if it is embed) and thus this style will be added always. So check for presence of window as well.